README.md

```diff
@@ -2,47 +2,117 @@
 
 ## Partner Study
 #### Answers to the following partner-study can be found [here](https://reactjs.org/docs/glossary.html)
-  * Describe the following, Single Page Application, Compilers, Bundlers, Elements, Components, JSX, Package Mangers, CDN, Props and State. 
-  * Write all answers in answers.md
+  * Describe the following, writing all answer in `answers.md`
+    - Single Page Application,
+    - Compilers, 
+    - Bundlers,
+    - Elements,
+    - Components,
+    - JSX,
+    - Package Mangers,
+    - CDN, 
+    - Props and State.
```

Comment:
The answer.md file has these listed numerically. I think it's better to save the screen real-estate here in the readme.md file. I don't think we need the code formatting for the file name, but I might specify:

   * Write your answers in the answers.md file.